deployment/hasura/metadata/databases/AerieMerlin/tables/public_activity_directive_changelog.yaml

```diff
@@ -0,0 +1,32 @@
+table:
+  name: activity_directive_changelog
+  schema: public
+object_relationships:
+- name: activity_directive
+  using:
+    manual_configuration:
+      remote_table:
+        name: activity_directive
+        schema: public
+      column_mapping:
+        plan_id: plan_id
+        activity_directive_id: id
+select_permissions:
+  - role: aerie_admin
```

Comment:
`aerie_admin` needs delete permissions on this table, and should probably be allowed to update the `changed_by` column